frontend/src/concepts/roleBinding/RoleBindingPermissionsTableRow.tsx

```diff
@@ -41,18 +43,20 @@ const ProjectSharingTableRow: React.FC<ProjectSharingTableRowProps> = ({
   onDelete,
 }) => {
   const [roleBindingName, setRoleBindingName] = React.useState(defaultValueName(obj));
-  const [roleBindingRoleRef, setRoleBindingRoleRef] = React.useState<ProjectSharingRoleType>(
-    defaultValueRole(obj),
-  );
+  const [roleBindingRoleRef, setRoleBindingRoleRef] =
+    React.useState<RoleBindingPermissionsRoleType>(defaultValueRole(obj));
   const [isLoading, setIsLoading] = React.useState(false);
   const createdDate = new Date(obj.metadata.creationTimestamp || '');
+  const isModelRegistry = obj.metadata.namespace === MODEL_REGISTRY_DEFAULT_NAMESPACE;
+  const disableActions =
+    isModelRegistry && obj.metadata.name === `${obj.metadata.labels?.app}-users`;
```

Comment:
Same here, can we remove `isModelRegistry` and references to `MODEL_REGISTRY_DEFAULT_NAMESPACE` and instead lift out props that describe what the actual effects of the logic are? Like for example `RoleBindingPermissionsTableRow` could have a `hasPermissionSelection: boolean` prop and maybe a `disableActionsForRbApp: string`... although...

Are we sure this is the right `disableActions` logic? We want to disable actions for the rolebinding associated with the default group. Maybe we instead want to create a prop like `disableActionsForSubject?: RoleBindingSubject`, and then the logic here can be something like:

```ts
  const disableActions =
    disableActionsForSubject &&
    !!obj.subjects.find(
      ({ kind, name }) =>
        kind === disableActionsForSubject.kind && name === disableActionsForSubject.name,
    );
```

and then you'd pass down the prop:

```ts
  disableActionsForSubject={{
    kind: 'Group',
    apiGroup: 'rbac.authorization.k8s.io',
    name: `${modelRegistry.metadata.name}-users`,
  }}
```

For both of these examples, you'd need to lift out these new props `hasPermissionSelection` and `disableActionsForSubject` (or whatever you call them) a few layers, through `RoleBindingPermissionsTable`, `RoleBindingPermissionsTableSection`, `RoleBindingPermissions`, until you get to `ModelRegistriesPermissions` where you can pass their values specific to MR. The goal here is to not have any imports from `~/concepts/modelRegistry` inside `~/concepts/roleBinding` if possible. It'll be more extensible/maintainable that way.

Reply:
hm.. with latest change i'm putting "This group is created by default. You can add users to this group via the API", so maybe I need to pass an `isDefaultGroup` prop of some sort

Reply:
Yeah maybe instead of `disableActionsForSubject` it's just `defaultGroupName` as an optional prop, and if you pass a default group it disables actions on that group and also adds that help text to it.

Reply:
We could open a tech debt issue for this too if you can't get to it today, just because I guess technically the auto-generated RoleBinding has the name `${obj.metadata.labels?.app}-users` in addition to the group itself so this logic will work.